solr/solrj/src/java/org/apache/solr/common/util/DOMUtil.java

```diff
@@ -37,9 +39,24 @@
 
   public static final String XML_RESERVED_PREFIX = "xml";
 
+  public static final Set<String>  NL_TAGS = Set.of("str", "int","long","float","double","bool");
+
+
   public static Map<String,String> toMap(NamedNodeMap attrs) {
     return toMapExcept(attrs);
   }
+  public static Map<String,String> toMap(ConfigNode node) {
+    return toMapExcept(node);
+  }
+  public static Map<String,String> toMapExcept(ConfigNode node, String... exclusions) {
```

Comment:
I wish this were a Set instead of an array, but I see the parallel to the other implementation of this.